packages/nextjs/app/builders/0x972d1be405DcFcD32DD95743934693B673e3F2B0/page.tsx

```diff
@@ -0,0 +1,102 @@
+import Image from "next/image";
+import { NextPage } from "next";
+import { FaEnvelope, FaGithub, FaMapMarkerAlt } from "react-icons/fa";
+
+interface ProfileData {
+  name: string;
+  title: string;
+  bio: string;
+  avatar: string;
+  location: string;
+  email: string;
+  social: {
+    github: string;
+    twitter: string;
+  };
+}
+
+const MillburnBuilderPage: NextPage = () => {
+  const profile: ProfileData = {
+    name: "Developer Millburn",
+    title: "Smart Contract Developer",
+    bio: "I'm a blockchain developer, versitle in writing smart contract using Solidity, Hardhat, Foundry, Remix, EthersJs.",
+    avatar: "/blackcat.jpg",
+    location: "Kogi State, Nigeria",
+    email: "millburncrack@gmail.com",
+    social: {
+      github: "https://github.com/MillburnCrackDev",
+      twitter: "https://x.com/Millburn",
+    },
+  };
+
+  // Social links with direct href values as recommended by maintainer
+  const socialLinks = [
+    {
+      href: profile.social.github,
+      icon: FaGithub,
+      lightHoverColor: "hover:text-gray-900",
+      darkHoverColor: "dark:hover:text-gray-200",
+    },
+    {
+      href: `mailto:${profile.email}`,
+      icon: FaEnvelope,
+      lightHoverColor: "hover:text-red-500",
+      darkHoverColor: "dark:hover:text-red-400",
+    },
+  ];
```

Comment:
Since these are all constant, it makes sense to place them outside of the component. So they're created only once at module initialization, rather than every render.